apps/discord/app.py

```
@@ -0,0 +1,344 @@
+
+
+@app.function()
+@modal.concurrent(max_inputs=1000)
+async def fetch_api() -> str:
+    """Fetch random API data (legacy function for backward compatibility)."""
+    handler = CommandHandler()
+    return await handler._fetch_api_data()
+
+
+@app.local_entrypoint()
+def test_fetch_api():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_fetch_api`

Add return type annotation: `None`

(ANN201)